<launch>
	<arg name="p_term" default="0.001" />
	<arg name="i_term" default="0.0" />
	<arg name="d_term" default="0.0" />
	<arg name="flip_linear" default="false" />
	<arg name="flip_angular" default="false" />
	<arg name="using_gps" default="false" />
	<arg name="using_lidar" default="false" />
	<arg name="using_camera" default="false" />
	<arg name="max_speed" default="0.2" />
	<arg name="max_turn" default="0.15" />

	<node pkg="heading_control" type="heading_control_node" name="heading_control" output="screen">
		<param name="kP" value="$(arg p_term)" />
		<param name="kI" value="$(arg i_term)" />
		<param name="kD" value="$(arg d_term)" />
		<param name="flip_fb" value="$(arg flip_linear)" />
		<param name="flip_rl" value="$(arg flip_angular)" />
		<param name="enable_gps" value="$(arg using_gps)" />
		<param name="enable_lidar" value="$(arg using_lidar)" />
		<param name="enable_camera" value="$(arg using_camera)" />
		<param name="max_linear" value="$(arg max_speed)" />
		<param name="max_angular" value="$(arg max_turn)" />
		<param name="base_frame" value="ohm_base_link" />
		<param name="reference_frame" value="world" />
	</node>
</launch>
